src/features/Search/components/result/index.js

```diff
@@ -0,0 +1,18 @@
+import * as styles from './index.module.css';
+
+export const SearchResult = (props) => {
```

Comment:
Rule of thumb, keep components as simple as possible and as stupid as possible, try to build functionality on a higher level, feature-level, and always think about the reusability of a component.

For better readability, I try to separate components to smaller units, and then when building the feature it's usually easier to read the code